devbuild.sh

```diff
@@ -228,10 +237,10 @@ if [ "${VERBOSE}" = true ]; then
   MAKE_SETTINGS="${MAKE_SETTINGS} VERBOSE=1"
 fi
 
-# source the environment file for this platform/compiler combination, then build the code
-printf "... Source ENV_FILE and create BUILD directory ...\n"
-module use ${SRC_DIR}/env
-. ${ENV_FILE}
+# source the module file for this platform/compiler combination, then build the code
+printf "... Load MODULE_FILE and create BUILD directory ...\n"
+module use ${SRC_DIR}/modulefiles
```

Comment:
@danielabdi-noaa, the build process doesn't go further without "module purge" before module load. If this is an issue only on wcoss_dell_p3, can you add if statement for wcoss_dell_p3 and put "module purge" there?

Reply:
@danielabdi-noaa, I confirmed that the build process was completed successfully on wcoss_dell_p3, hera, and orion.